specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -149,3 +166,15 @@ interface TextBlocklists {
     ListQueryParametersTrait<StandardListQueryParameters>
   >;
 }
+
+@added(Versions.v2024_02_15_Preview)
+interface TextGroundednessDetectionOperations {
+  @summary("Detect Groundedness")
+  @doc("A synchronous API for the analysis of language model outputs to determine alignment with user-provided information or identify fictional content.")
+  @route("/text:detectGroundedness")
+  @post
+  detectGroundednessOptions is Azure.Core.RpcOperation<
```

Comment:
```suggestion
  detectGroundedness is Azure.Core.RpcOperation<
```

Reply:
is it possible to change the name here? it's duplicated with the model definition here https://github.com/Azure/azure-rest-api-specs/pull/27654/files#diff-62e682bbeb17329bdf304301b4aac7e89a59392e681c3853a2a8f1d094c48419R282